javacutil/src/org/checkerframework/javacutil/TypeAnnotationUtils.java

```diff
@@ -65,6 +65,25 @@ public static boolean isTypeCompoundContained(Types types, List<TypeCompound> li
         return false;
     }
 
+    /**
+     * Check whether a TypeCompound with the same name and position is contained in a list of TypeCompounds.
+     *
```

Comment:
Some additional contrasting with the existing method would be good.
Add a note to the existing method.